spec/features/user_cant_sign_in_spec.rb

```diff
@@ -83,6 +83,45 @@
 
       screenshot!
     end
+
+    scenario "user tries to login with unconfirmed email" do
+      user = create_user 'user'
+      email = create_email_address_for user, 'user@example.com', confirmation_code='not yet'
+
+      complete_login_username_or_email_screen('user@example.com')
+
+      expect(page).to have_content(t(:'sessions.start.unknown_email'))
+    end
+
+    scenario "oauth user tries to login with unconfirmed email" do
+      user = create_user 'user'
+      email = create_email_address_for user, 'user@example.com', confirmation_code='not yet'
+      FactoryGirl.create :authentication, provider: 'google_oauth2', user: user
+      complete_login_username_or_email_screen('user')
+
+      with_omniauth_test_mode(email: 'user@example.com') do
```

Comment:
This test is really testing that the uid provided by google does not match the uid of the authentication object created above (since you did not pass a uid to with_omniauth_test_mode, it uses a random one). Because of this, it is a dupe of https://github.com/openstax/accounts/blob/ff73ded343e11f578a3bfbf22436b9c6dbbe104a/spec/features/user_cant_sign_in_spec.rb#L205